app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -388,10 +437,9 @@ private void launchAlert(String title, String message){
     private boolean checkAccount(){
         Account currentAccount = sessionManager.getCurrentAccount();
         if (currentAccount == null) {
-        Timber.d("Current account is null");
```

Comment:
Why did you modify this line and remove Timber logs, it is valid.

Reply:
I see no need to modify this part at all

Reply:
I thought you have mentioned to remove all logs and comment statements but if these things are valid then i'll put them back